cPP_DSC.adoc

```diff
@@ -356,58 +356,58 @@ One way to categorize keys is by the cryptographic functions they are allowed to
+Certificates contain either signed public keys or some sort of Authorization Data. Signature keys come in several varieties: signature generation keys, which contain a private key for signing (and maybe also the public key for verification) and signature verification keys, which contain only the public verification key and do not contain the private key (and thus cannot perform a signing function).
 
 Authorization Data may have an arbitrary length of bits or bytes and may contain arbitrary or non-arbitrary values of bits or bytes.
 
-Seeds have a special place in this Key Reference Model. Manufacturers, owners, and users of the DSC can use permanent seeds to create root keys. Manufacturers have good reasons to use seeds to derive Root Keys and other items in the Key Reference Model. These include:
+Seeds have a special place in this Key Reference Model. Manufacturers, owners, and users of the DSC can use permanent seeds to create Root Keys. Manufacturers have good reasons to use seeds to derive Root Keys and other items in the Key Reference Model. These include:
 
 * Seeds take less space to store than certain asymmetric keys for given desired cryptographic strengths.
-* Having seeds that are unique per DSC enhances the chance that the same key derivation function on different DSCs will yield unique keys.
+* Having seeds that are unique per DSC increases the probability that the same key derivation function on different DSCs will yield unique keys.
 
-<<ExampleKeyHierarchy>> contains an example of a hierarchy of keys where each lower-level key is wrapped by a higher-level key that is connected to it. The Firmware Signature Key and the Root Encryption Key are examples of Root Keys. The Intermediate Wrapping Key is an example of an Intermediate Key. The Software Signature Key, the File Encryption Key, and the Streaming Movie Authorization Token are examples of Leaf Objects. <<ExampleKeyHierarchy>> serves as an illustration of key hierarchies; other configurations are possible.
+<<ExampleKeyHierarchy>> contains an example of a hierarchy of keys where each lower-level key is wrapped by a higher-level key that is connected to it. The Root Encryption Key is an example of a Root Key. The numbered Key Encryption Keys are examples of Intermediate Keys. The Data Encryption Keys and Stored Keys are examples of Leaf Objects. <<ExampleKeyHierarchy>> serves as an illustration of key hierarchies; other configurations are possible.
```

Comment:
This sentence was not consistent with the diagram. Was the diagram updated?

Reply:
I don't think so, I expect the text was meant to be more broad, but since it references the example, I think it is fine to refine the description. I don't know if we will want to have those other examples somewhere, I'm not sure, but to me this is fine.